mpxe/integration_tests/test_babydriver.py

```diff
@@ -0,0 +1,49 @@
+import sys
+sys.path.append('projects/baby_driver/scripts')
```

Comment:
I am currently looking on the web if there is any other way of including this path but for now this works even though it s very hacky.